data/governments.txt

```diff
@@ -254,6 +254,32 @@ government "Drak (Incipias)"
 		"Conlatio" 1
 		"Incipias Civilian" 1
 
+government "Drak (Nenia)"
+	"display name" "Drak"
+	swizzle 0
+	color 1 1 1
+	"player reputation" 1
+	"attitude toward"
+		"Indigenous Lifeform" 1
+		"Ember Waste" 1
+	illegals
+		ignore "Nerve Gas"
```

Comment:
I'm afraid this won't work as you want, because even vanilla has at least one more illegal outfit (the Korath defense station) and plugin authors would have to add every illegal item here. I'll open a code PR soon to help with this.

Reply:
Added in the "Microbot Defense Station" -- should this hang tight until the code PR or do you think ignoring vanilla illegal outfits would be sufficient?